server/app.js

```diff
@@ -1,15 +1,148 @@
 const express = require("express");
-
+const mongoose = require("mongoose");
 const app = express();
+const cors = require("cors");
+const Students = require("./models/student");
+const Course = require("./models/course");
+const { restart } = require("nodemon");
 
 app.use(express.json());
+app.use(cors());
+
+//Get Endpoints
+
+//get a single course
+app.get("/course/:courseId", (req, res) => {
+  const { id } = req.params;
+  Course.findById(id).then((courses) => {
+    res.status(200);
+    res.json(course);
+  });
+});
+
+//get all courses
+app.get("/course", (req, res) => {
+  Course.find().then((course) => {
+    res.status(200);
+    res.json(course);
+  });
+});
+
+//get all Students in a course
+app.get("/courses/:courseId/students", (req, res) => {
+  Students.find().then((students) => {
+    res.status(200);
+    res.json(students);
+  });
```

Comment:
Technically this will always return all students in the database, regardless of their assigned course. You'd have to filter by what `courseId` the students have assigned to them :)